src/Microsoft.TemplateSearch.Common/Microsoft.TemplateSearch.Common.csproj

```diff
@@ -12,6 +14,22 @@
 
   <ItemGroup>
     <ProjectReference Include="$(SrcDir)Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
```

Comment:
https://github.com/dotnet/templating/blob/bc4a41245c034f4f3b7fc5056bdcba9d6da591e5/src/Microsoft.TemplateSearch.Common/TemplateDiscoveryMetadata/LegacySearchCacheReader.cs#L35-L46

we need it for dynamic type conversions here. Since it's a temp solution i didn't add it to Versions.props file. I will add comment here and in reader not to forget to remove it when legacy reader is dead.